apps/vr-tests/src/stories/Button.stories.tsx

```diff
@@ -170,6 +170,9 @@ storiesOf('Button Split', module)
   .addStory('Primary Disabled', () => (
     <DefaultButton {...commandProps} primary={true} disabled={true} split={true} />
   ))
+  .addStory('Primary with Primary Action Disabled', () => (
+    <DefaultButton {...commandProps} primaryDisabled={true} primary={true} split={true} />
```

Comment:
Can we add a default variant with primary disabled here?

Reply:
@GitHubMaxwell if you add a default variant story make sure you bug Ben 😄 to get an approval as soon as the CI is done.